scripts/crowbar-testbuild.yaml

```diff
@@ -11,6 +11,10 @@ branches:
     bs_project:  Devel:Cloud:7:Staging
     bs_repo:     SLE_12_SP2
     cloudsource: develcloud7
+  master:
```

Comment:
I think both needs to be done at the same time, coordinaed with the git branching. otherwise pull requests are gated against the wrong product.